### ISSUE TITLE
New HMM voice sounds too noisy

### ISSUE THREAD

**Issue by kosmasK** [marytts/marytts#635]
Dear MaryTTS community,

I am trying to build a new voice with MaryTTS 5.2 from scratch by using an audiobook. I have more or less one hour of voice and it got transcribed to 1400 utterances (wav, text). I followed all the steps as they were described at HMMVoiceCreation wiki. I kept all the configurations to the default with only exception that I decreased the number of HMM states from 5 to 4, since I was facing same errors as in the issue number #134 . I have compiled the voice with success and pasted the xml in the “installed” folder and the jar in the “lib” folder, under the “target/marrytts-5.2” folder of my build. 
However, the sonic result of my voice is vary poor and noisy.. My hypothesis was that, for some reason, the aperiodic component in the synthesis level is too prominent in comparison to the other components, as it is presented in the following compressed examples:

[Voice_Examples.zip](https://github.com/marytts/marytts/files/581785/Voice_Examples.zip)

Also you can listen an energy drop in the “long.wav”. Possible answer can be the one described in the issue number #177 .

To sum up.. What is wrong with my new voice? 
Would you judge that the audible defects indicate that there’s something going wrong in the synthesis level? 
In the quantity and/or quality of the collected data? 
Maybe in the configuration parameters? 
At which stage of the procedure do you think I should focus?
In few words, why does it sound like this???

I would appreciate it if you can provide me with some hints!
Looking forward for your response!

Cheers!

**Comment by nshmyrev** [comment]
You need to synthesize the sample from db for debugging.

You need to study predicted f0 value to see if values are reasonable. If values are not reasonable, you need to study f0 extracted from the training samples in order to see if there were issues.

If you work with English, it's recommended to train from slt database first.

> I kept all the configurations to the default with only exception that I decreased the number of HMM states from 5 to 4, since I was facing same errors as in the issue number #134 .

This was a bad idea, you should ensure phonemes are covered properly instead.

**Comment by jesinj** [comment]
Hello,

Was this problem solved? I am also facing a similar issue with the voice I built.

I did build a female voice, and it worked well. Now when I built a male voice, it sounds all muffled, like the speech samples you have posted.

If you could solve the issues, do provide me the details.

Regards

**Comment by seblemaguer** [comment]
Hello,

the problem is the voicing/unvoicing property (according to the sample I hear). You should check the f0 boundaries and the values you get before actually go through the model training process
